autotest/ogr/ogr_gml_citygml_read.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env pytest
+# -*- coding: utf-8 -*-
+###############################################################################
+# $Id$
+#
+# Project:  GDAL/OGR Test Suite
+# Purpose:  GML Reading Driver for CityGML testing.
+# Author:   mugwort_rc <mugwort.rc@gmail.com>
+#
+###############################################################################
+# Copyright (c) 2021, mugwort_rc <mugwort.rc@gmail.com>
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+# OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+# DEALINGS IN THE SOFTWARE.
+###############################################################################
+
+
+
+import ogrtest
+from osgeo import gdal
+from osgeo import ogr
+from osgeo import osr
+import pytest
+
+
+###############################################################################
+# Test reading CityGML files
+###############################################################################
+
+_citygml_dir = 'data/gml_citygml/'
+
+###############################################################################
+# Test reading CityGML of Project PLATEAU
+
+def test_gml_read_compound_crs_lat_long():
```

Comment:
and I would just move that test in the existing autotest/ogr/ogr_gml_read.py

Reply:
OK. I moved it.